packages/editor/src/components/block-mover/style.scss

```diff
@@ -93,6 +93,7 @@
 .editor-block-mover__control-drag-handle {
 	cursor: move; // Fallback for IE/Edge < 14
 	cursor: grab;
+	fill: currentColor;
```

Comment:
Some people asked about "dark theme" support for the drag handle, will this fix it?

Reply:
Can you clarify "dark theme" support? Because there's one issue for providing an actual dark theme for Gutenberg. 

But yes, if paired with <a href="https://wordpress.org/gutenberg/handbook/extensibility/theme-support/#add-the-stylesheet">add_theme_support( 'dark-editor-style' );</a> and a dark editor style, this handle would be inverted.

Reply:
Yes, I believe it's about `add_theme_support( 'dark-editor-style' );` I just saw this here https://wordpress.slack.com/archives/C02QB2JS7/p1537090223000100